openapi/components/schemas/StorefrontQuoteChangeOrder.yaml

```
@@ -0,0 +1,474 @@
+type: object
+required:
+  - websiteId
+  - customerId
+  - items
+properties:
+  id:
+    readOnly: true
+    description: ID of the quote.
+    type: string
+    maxLength: 50
+    example: qt_0YV7DES3WPC5J8JD8QTVNZBZNZ
+  type:
+    description: Specifies the type of the quote. A quote can be a subscription or a one-time purchase.
+    type: string
+    readOnly: true
+    enum:
+      - subscription-order
+      - one-time-order
+  action:
+    description: |-
+      Action of the quote for change order items.
+      On accept quote interim invoice is issued and linked to `invoiceId`.
+      After the invoice is paid changed items are applied to an order with `orderId`.
+    type: string
+    enum:
+      - change
+    example: change
+  acceptanceConditions:
+    type: array
+    writeOnly: true
+    description: If conditions are fulfilled the quote is automatically accepted.
```

suggestion: for readability
```suggestion
    description: Conditions that must be met for the quote to be accepted.
```